tests/index/test_index_search_meilisearch.py

```diff
@@ -473,6 +473,20 @@ def test_show_ranking_score_details(index_with_documents):
     assert response["hits"][0]["_rankingScoreDetails"]["words"]["score"] == 1
 
 
+def test_show_ranking_score(index_with_documents):
+    """Tests search with show ranking score"""
+    response = index_with_documents().search(
+        "man loves",
+        {"showRankingScore": True},
+    )
+
+    print(response["hits"][0]["_rankingScore"])
```

Comment:
I assume this print was for helping set up the test? It can be removed.

Reply:
sorry, forgot to remove this